packages/contract-utils/upgradeable/src/upgradeable.rs

```diff
@@ -0,0 +1,112 @@
+use soroban_sdk::{contractclient, contracterror, Address, BytesN, Env, FromVal, Val};
```

Comment:
should we emit an event for `upgrade` and `downgrade`? 
It may be useful to have this info available on block explorers, wdyt?

Reply:
There's a [system event emitted](https://github.com/stellar/rs-soroban-env/blob/e3feadbc570b9fb88769dbea26e83a52f48cca2e/soroban-env-host/src/events/system_events.rs#L16) when updating a contract executable code, making additional event unnecessary here.

Reply:
good to know, thanks!